.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please don't commit files with personal settings

Reply:
These settings were inferred form the project. I would consider them project settings.

Here is example why I think it is important to save project settings files. This project diverged from the default convention of using `test*.py` pattern for tests files. It requires custom patten to be passed. I had to look it up in `unittest` documentation. I want to make it easier for others. One option would be to extend the `CONTRIBUTING.md` file and include instructions on how to configure our environment. Another is to save project settings files. I chose the latter.

```
>python -m unittest discover

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```
```
>python -m unittest discover --pattern "*_test.py" -s tests
...
----------------------------------------------------------------------
Ran 3 tests in 0.001s

OK
```

Reply:
I prefer to change the convention to the default one. You can just rename masking_test.py to test_masking.py

Reply:
Thanks! I've just renamed masking_test and removed settings file.